astor/rtrip.py

```diff
@@ -85,6 +85,10 @@
 dsttree = 'tmp_rtrip'
 
 
+def out_prep(s, pre_encoded=(type(b'') is type(''))):
```

Comment:
Could you also add a TODO comment so we can remove this workaround when we drop legacy Python support in the future?